packages/dustkit/src/apps/getProgramAppConfigUrl.ts

```diff
@@ -26,7 +26,10 @@ export async function getProgramAppConfigUrl({
       systemId: program,
       abi: appAbi,
       functionName: "appConfigURI",
-      args: [entity ?? "0x"],
+      args: [
+        entity ??
+          "0x0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
oops, previous comment was in a commit rather than PR

could use `zeroHash` from viem for this

Reply:
updated! Didnt know this existed, can replace the same string in the client too

Reply:
there's also `zeroAddress`